kitsune/customercare/zendesk.py

```diff
@@ -94,11 +87,11 @@ def create_ticket(
         ticket = Ticket(
             subject=subject,
             comment={"body": description},
-            ticket_form_id=TICKET_FORM_ID,
+            ticket_form_id=settings.ZENDESK_TICKET_FORM_ID,
             custom_fields=[
-                {"id": PRODUCT_FIELD_ID, "value": product},
-                {"id": CATEGORY_FIELD_ID, "value": category},
-                {"id": OS_FIELD_ID, "value": os},
+                {"id": settings.ZENDESK_VPN_FIELD_ID, "value": product},
```

Comment:
this should be called `ZENDESK_PRODUCT_FIELD_ID`

Reply:
what if we add another product though?

Reply:
this is the ID for the product field, not VPN specifically - another product would use the same field, with a different value